content/Watsonx/WatsonxData/205.md

```diff
@@ -0,0 +1,250 @@
+---
+title: '205: Ingesting Data'
+timeToComplete: 20
+updated: 2024-02-02
+---
+
+## Ingesting Data
+
+Data ingestion is the process of importing and loading data into IBM® watsonx.data. One 
+approach is to use the Ingestion jobs tab from the Data manager page to load data securely and 
+easily into watsonx.data console (**NOTE**: This is only available in Standard Edition). Data 
+ingestion in watsonx.data console or in the ibm-lh command line utility supports CSV and 
+Parquet formats. The files that are ingested in a single ingestion command must be of the same 
+format type and schema. watsonx.data auto discovers the schema based on the source file 
+being ingested.
+
+You can also load or ingest local data files or ingest data from external sources using the ibm-lh
+tool. The ibm-lh command-line utility is a terminal-based interface that is designed to facilitate 
+interaction with watsonx.data resources. 
+
+This lab uses the ingest tool (ibm-lh) alongside the IBM watsonx.data Developer Edition that is 
+running in this lab. The Ingest tool is a separate install and currently needs to be downloaded 
+after IBM watsonx.data is started. The lab image (Developer Edition) contains a copy of this code 
+so you will not need to download it.
+
+
+In this section you will connect to your watsonx.data environment. The watsonx.data server 
+automatically starts all services. To check the status of the server, run the following commands
+using the instructions provided in **section 4.3. Command Line Access** of the “Prerequisites & 
```

Comment:
Provide the reference of the lab in the VEST site, not from the original content. In this case, you can refer to Lab 1: Environment Setup. Also, remove section 4.3.

Reply:
Changed